src/util/db.ts

```diff
@@ -48,11 +48,15 @@ export function createInstance(config: ConnectionConfig): Knex {
     throw new Error('The provided connection already contains a connection instance.');
   }
 
-  const { host, database } = config.connection as any;
+  const { host, server, database, userName } = config.connection as any;
 
-  log(`Connecting to database: ${host}/${database}`);
+  log(`Connecting to database: ${server ? server : host}/${database}`);
 
-  return Knex({
+  if (config.client === 'mssql' && (!server || !userName)) {
+    throw new Error(`Db client ${config.client} requires key "server" and "userName".`);
```

Comment:
Hmm I think we should simplify the API by still using `user` and `host` but just internally use `host` and `userName`. Meaning, we should let sync-db users still pass `host` and `user` and we internally map it to `server` and `userName` when passing it to knex?

What do you think? @kshitish182

Reply:
Yes, I think we can handle it this way. Will make the changes here.